File: src/main/java/com/summer/lijiahao/utils/OpenDictionaryAction.java
package com.summer.lijiahao.utils;

import com.intellij.ide.BrowserUtil;
import com.intellij.openapi.actionSystem.AnActionEvent;
import com.summer.lijiahao.abs.AbstractAnAction;

public class OpenDictionaryAction extends AbstractAnAction {
    private final static String URL = "https://nccdev.yonyou.com/datadict/datadict-2111/index.html";

    @Override
    public void doAction(AnActionEvent event) {
        BrowserUtil.open(URL);
    }
}

File: src/main/kotlin/com/summer/lijiahao/nccdevtool/menu/action/OpenDictionaryAction.kt
package com.summer.lijiahao.nccdevtool.menu.action

import com.intellij.ide.BrowserUtil
import com.intellij.openapi.actionSystem.AnActionEvent
import com.summer.lijiahao.nccdevtool.base.action.AbstractAnAction

class OpenDictionaryAction : AbstractAnAction() {
    private val URL = "https://nccdev.yonyou.com/datadict/datadict-2111/index.html"

    override fun doAction(event: AnActionEvent) {
        BrowserUtil.open(URL)
    }
}